_includes/nav.html

```diff
@@ -2,7 +2,7 @@
   <div class="announcement" role="alert">
     <div class="container">
       <p>
-        Our Networks 2018 is over! Check out videos of <a href="/conference/recorded-talks/">recorded talks</a>
+        Our Networks 2018 has ended! Check out <a href="/conference/recorded-talks/">videos of recorded talks</a> or <a href="https://ournetworks.ca">what we've been up to since</a>
```

Comment:
Is the intent with the `ournetworks.ca` link that it'll only be shown on `2018.ournetworks.ca`? Checking that we want it to show on `ournetworks.ca` since this is going to deploy from `master` on a successful build.